addons/project/data/mail_template_data.xml

```diff
@@ -123,6 +123,19 @@
 </div>
         </template>
 
+        <template id="project_mail_notification_layout" inherit_id="mail.mail_notification_layout" primary="True">
+            <xpath expr="//div[@summary='o_mail_notification']" position="replace">
+                <t t-set="model" t-value="record._name"/>
+                <t t-if="model == 'project.task'">
+                <ul style="margin-left: 12px; padding-bottom: 5px; margin-bottom: 20px;">
+                    <li style="font-size: 13px;">Project: <t t-out="record.project_id.name"/> &#8594; Task Created</li>
+                </ul>
+                <hr width="100%"
+                style="background-color: rgb(204,204,204); border: medium none; clear: both; display: block; font-size: 0px; min-height: 1px; line-height: 0; margin: 10px 0px;"/>
+                </t>
```

Comment:
```suggestion
                <t t-if="record._name == 'project.task'">
                    <ul style="margin-left: 12px; padding-bottom: 5px; margin-bottom: 20px;">
                        <li style="font-size: 13px;">Project: <t t-out="record.project_id.name"/> &#8594; Task Created</li>
                    </ul>
                    <hr width="100%"
                        style="background-color: rgb(204,204,204); border: medium none; clear: both; display: block; font-size: 0px; min-height: 1px; line-height: 0; margin: 10px 0px;"
                    />
                </t>
```
- Was the `t-set` necessary? We only use it once.
- Indented what is inside of the `t-if`.
- Align `style` attr with `width` attr.
- Shouldn't the the color be dynamic?

Reply:
Yes, color can be dynamic but I think for separator these background-color is more suitable.